netbox/dcim/filtersets.py

```diff
@@ -1733,6 +1733,17 @@ class InterfaceFilterSet(
         queryset=WirelessLink.objects.all(),
         label=_('Wireless link')
     )
+    vlan_translation_policy_id = django_filters.ModelMultipleChoiceFilter(
+        field_name='vlan_translation_policy',
+        queryset=VLANTranslationPolicy.objects.all(),
+        label=_('VLAN Translation Policy (ID)'),
+    )
+    vlan_translation_policy_name = django_filters.ModelMultipleChoiceFilter(
```

Comment:
Be sure to follow the same convention as existing filters.
```suggestion
    vlan_translation_policy = django_filters.ModelMultipleChoiceFilter(
```